db/seeds.rb

```diff
@@ -53,6 +53,5 @@
     )
   end
 
-  subject_csv = Rails.root.join("config", "subjects.csv")
-  SubjectImporter.new(subject_csv).run
+  Rake::Task["subjects:import"].invoke("2017FA")
```

Comment:
Dividing up the months could work. Although, you are right that it is easy to run the rake task again with the correct term. Let's leave things as is for now. We can make a new PR for this and tackle during the end if there is time.